lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -84,6 +84,12 @@ def self.available_options
                                        verify_block: proc do |value|
                                          UI.user_error!("firebase_app_distribution: Couldn't find ipa file at path '#{value}'") unless File.exist?(value)
                                        end),
+          FastlaneCore::ConfigItem.new(key: :googleservice_info_plist_path,
```

Comment:
Is this parameter generally going to be used to change only the name of the file? Or will it be used to set the path relative to the project root?

If it's the former, can you please change "path" to "name" throughout the ConfigItem declaration and its uses?

If it's the latter, can you update the description to be "Path to your GoogleService-Info.plist file, relative to the root of your Xcode project"

Reply:
I updated the description to _"Path to your GoogleService-Info.plist file, relative to the root of your Xcode project"_, because I find it to describe a more flexible usage IMHO, even though in the company I work at we use it to change the name of the file only

Reply:
Sounds good!